org_fedora_oscap/common.py

```diff
@@ -54,6 +55,13 @@
 
 SSG_DIR = "/usr/share/xml/scap/ssg/content/"
 SSG_XCCDF = "ssg-rhel7-xccdf.xml"
+if product.productName.lower() != 'anaconda':
+    if product.productName.lower() == 'fedora':
+        SSG_XCCDF = "ssg-fedora-xccdf.xml"
+    elif product.productName.lower() == 'rhel':
+        SSG_XCCDF = "ssg-rhel%s-xccdf.xml" %(product.productVersion)
```

Comment:
This is a useless branch, it does the same as the branch below it

Reply:
@mpreisler should we just do this:

```
if product.productName.lower() != 'anaconda':
    if product.productName.lower() == 'fedora':
        SSG_XCCDF = "ssg-fedora-xccdf.xml"
    else:
        SSG_XCCDF = "ssg-%s%s-xccdf.xml" %(product.productName.lower(), product.productVersion)
```

Reply:
Yeah and please make it pep8 compliant.